examples/policies/kubernetes/namespace/demo-pods.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
```

Comment:
Is this file referenced in the documentation?

Reply:
I meant to, will fix.